api/include/opentelemetry/context/runtime_context.h

```diff
@@ -47,6 +47,38 @@ class RuntimeContext
 
   static RuntimeContext *context_handler_;
 
+  // Sets the Key and Value into the passed in context or if the context* is null,
+  // the RuntimeContext.
+  static Context SetValue(nostd::string_view key, ContextValue value, Context *context)
```

Comment:
Consider providing a default nullptr value for context in both of these functions, which would make it easier to just call

RuntimeContext::SetValue(key, value);

Reply:
Added default nullptr.